libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/extensions/NullEx.kt

```diff
@@ -0,0 +1,80 @@
+package com.mapbox.services.android.navigation.v5.utils.extensions
```

Comment:
Could we rename the file into `NullExtensions.kt`? `NullEx.kt` is kinda confusing IMO @RingerJK

Reply:
in my experience short `Ex` better. In feature we'll have smt like `ExternalNavigationManagerEx` better than `ExternalNavigationManagerExtensions` IMO. Also this name are hidden into Kotlin's project and only appear at left navigation bar. Let's keep just `Ex` suffix. wdyt?